examples/electricitymap/main.go

```diff
@@ -19,6 +19,10 @@ func main() {
 	c := provider.ElectricityMapConfig{
 		Token: token,
 	}
+	url := os.Getenv("ELECTRICITY_MAP_API_URL")
+	if url != "" {
```

Comment:
This is question about Go more than anything else - does `os.Getenv` always default to an empty string if there isn't the named environment variable with a given key name? Or would it throw an exception / error if not defined?

I'm aware this is an example, so it needn't be super idiomatic or production grade.

Reply:
Yes `os.Getenv` just returns a string https://pkg.go.dev/os#Getenv 

If the env var is not set then it will default to `https://api.electricitymap.org/v3`. However since there are 3 different types of base URLs I think its best we ask the user to provide the URL.

I've changed the code and the example to error if the URL env var is not set.